### DISCUSSION TITLE
using mp3 encoder not tagged lamejs package

### DISCUSSION THREAD

**Discussion by guyYablonka** [RocketChat/fuselage#652] [Q&A] [answered]
hey, I wanted to know if there is a reason that mp3 encoder project uses dependency of lamejs from  "https://github.com/zhuker/lamejs.git#commit=564612b5b57336238a5920ba4c301b49f7cb2bab" and not the latetest tag "1.2.1"
thanks! :)

**Comment by tassoevan** [comment, marked as answer]
I believe that, at the time we have ported the code from https://github.com/RocketChat/Rocket.Chat to this repository, there was only version 1.2.0 and it was broken. Maybe now we can upgrade it.

**Reply by guyYablonka** [reply]
Got you, thanks
